admin/common_auth/views.py

```diff
@@ -77,6 +77,12 @@ def form_valid(self, form):
         prereg_admin_group = Group.objects.get(name='prereg_admin')
         for group in form.cleaned_data.get('group_perms'):
             osf_user.groups.add(group)
+            split = group.name.split('_')
+            group_type = split[0]
+            if group_type == 'reviews':
+                provider_id = split[1]
+                provider = PreprintProvider.load(provider_id)
+                provider.notification_subscriptions.get().add_user_to_subscription(osf_user, 'none')
```

Comment:
According to the user stories doc on the [moderator management epic ticket](https://openscience.atlassian.net/browse/IN-116):
>default is instantly